pipeline-model-definition/src/main/resources/org/jenkinsci/plugins/pipeline/modeldefinition/generator/AxesDirective/help-axis.html

```diff
@@ -0,0 +1,26 @@
+<!--
+  ~ The MIT License
+  ~
+  ~ Copyright (c) 2020, CloudBees, Inc.
+  ~
+  ~ Permission is hereby granted, free of charge, to any person obtaining a copy
+  ~ of this software and associated documentation files (the "Software"), to deal
+  ~ in the Software without restriction, including without limitation the rights
+  ~ to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+  ~ copies of the Software, and to permit persons to whom the Software is
+  ~ furnished to do so, subject to the following conditions:
+  ~
+  ~ The above copyright notice and this permission notice shall be included in
+  ~ all copies or substantial portions of the Software.
+  ~
+  ~ THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+  ~ IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+  ~ FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+  ~ AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+  ~ LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+  ~ OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+  ~ THE SOFTWARE.
+  -->
+<div>
+    Multiple options can be specified.
+</div>
```

Comment:
I don't know if there is a fix for this, because of the way the inline help is handled in the Jenkins UI. This comment is probably way beyond the scope of this PR. But I noticed it, so I thought I'd at least bring it up for discussion.

Here's a screen shot of me using `axes` --> `axis` in the generator.

<img width="1423" alt="image" src="https://user-images.githubusercontent.com/21689198/76096359-f4327480-5f93-11ea-9d1b-cea02750adb9.png">

I bring this up because it's impossible to tell which (?) icon lines up with which field in the page. For example, which of the three (?) icons shown below do I use to surface this help text?

<img width="1235" alt="image" src="https://user-images.githubusercontent.com/21689198/76096561-4d020d00-5f94-11ea-870c-619ad2dec299.png">

Things get more cluttered when I start opening more than one help at the same time:

<img width="1383" alt="image" src="https://user-images.githubusercontent.com/21689198/76096669-76229d80-5f94-11ea-80f9-f0689bb4a48f.png">

What would be really cool is if we could draw dotted lines around stuff according to what (?) button goes with a particular field. The Delete buttons already do this - if you hover over them, they draw a box around what I'm about to delete:

<img width="1237" alt="image" src="https://user-images.githubusercontent.com/21689198/76096925-d6194400-5f94-11ea-9b3b-371cdd2f5b95.png">